build/template-build-ios-automation.yaml

```diff
@@ -58,8 +58,8 @@ steps:
 - task: CopyFiles@2
   displayName: 'Copy artifacts to staging directory'
   inputs:
-    SourceFolder: tests/devapps/MauiApps/MauiAppBasic/artifacts
-    Contents: '**/*.*'
+    SourceFolder: tests/devapps/XForms/XForms.iOS/bin/iPhone/Release
```

Comment:
Do we have this (Release|Debug) in the parameters?

Reply:
No. We do not have the parameter. The test App built in the earlier step has Release configuration.